app/views/users/team.html.haml

```diff
@@ -5,7 +5,7 @@
     .column-two-thirds
       %h1.heading-large= t('views.users.team.page_title')
     .column-one-third
-      = link_to t('views.users.new.page_title'), new_user_invitation_path, class: "button align-with-heading"
+      = link_to t('views.users.new.page_title'), new_user_invitation_path('advanced'), class: "button align-with-heading"
```

Comment:
Aye but again, I think thats not a param passing thing, I think thats a cancancan `has this user the authority to see this button`. By leaving it as passing param rather having cancancan checking the current user you leave yourself open to security issues similar to how cancancan now decides who see's what navigation buttons on the home page.

Like it doesnt currently solve a purpose, this is the 2nd or 3rd time we've talked about this, its probably because it just shouldn't be there.